cmd/cli/fileshare_process.go

```diff
@@ -0,0 +1,11 @@
+//go:build fileshare_process
```

Comment:
GRPC fileshare process is implemented in the next commit. For now this file is not used anywhere thanks to conditional compilation.

Reply:
will `fileshare_process` be part of the `.env` file, added to `FEATURES`?

Reply:
As is, it's configured in `snapcraft/snap.yml. No reason to add it to `FEATURES` I think.